packages/web/tailwind.config.js

```diff
@@ -1,11 +1,11 @@
 module.exports = {
-  mode: 'jit',
```

Comment:
Why did you remove jit? @heybereket did you approve this?

Reply:
> Why did you remove jit? @heybereket did you approve this?

A day ago yes, but it worked fine, so we decided to keep it.